static/app/views/replays/replaySearchBar.tsx

```diff
@@ -121,6 +123,20 @@ function ReplaySearchBar(props: Props) {
       maxMenuHeight={500}
       hasRecentSearches
       fieldDefinitionGetter={getReplayFieldDefinition}
+      onSearch={(query: string) => {
+        props?.onSearch?.(query);
```

Comment:
props shouldn't need the `?`

```suggestion
        props.onSearch?.(query);
```